.github/workflows/azureml-unit-tests.yml

```diff
@@ -23,8 +23,11 @@ on:
       # Unit tests will be run only when there are changes in the
       # unit tests related code including:
       - examples/**
+      - '!examples/**/*.md'
```

Comment:
what you are saying is that this fix avoids to trigger the unit tests that if I modify this file https://github.com/microsoft/recommenders/blob/main/examples/00_quick_start/README.md?

Reply:
Yes